### ISSUE TITLE
Deselect All does not provide screen reader feedback

### ISSUE THREAD

**Issue by jhung** [platform-coop-toolkit/coop-library#131]
**Describe the bug**
When activating "Deselect all", there is a lack of screen reader feedback indicating filters have been unchecked.

**To Reproduce**
1. Turn on a screen reader (i.e. NVDA)
2. Go to 'Explore'
3. Check 'Cities' and 'Commons' filters
4. Activate "Deselect All"
5. Notice the lack of feedback given through the screen reader.

**Expected behavior**
When activating Deselect All, the screen reader should indicate the result of the action by reporting "X filters unchecked", or "All filters unchecked".

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Firefox
 - Version 72.0.1
